kotlin/query-features-with-arcade-expression/README.md

```diff
@@ -0,0 +1,57 @@
+# Query features with Arcade expression
+
+Query features on a map using an Arcade expression.
+
+![QueryFeaturesWithArcadeExpression](query-features-with-arcade-expression.png)
+
+## Use case
+
+Arcade is a portable, lightweight, and secure expression language used to create custom content in ArcGIS applications. Like other expression languages, it can perform mathematical calculations, manipulate text, and evaluate logical statements. It also supports multi-statement expressions, variables, and flow control statements. What makes Arcade particularly unique when compared to other expression and scripting languages is its inclusion of feature and geometry data types. This sample uses an Arcade expression to query the number of crimes in a neighborhood in the last 60 days.
+
+## How to use the sample
+
+Tap on any neighborhood to see the number of crimes in the last 60 days in a callout.
+
+## How it works
+
+1. Create a `PortalItem` using the URL and ID.
+2. Create a `Map` using the portal item.
+3. Set the visibility of all the layers to false, except for the layer named "RPD Beats  - City_Beats_Border_1128-4500".
```

Comment:
Maybe we should just make a feature layer from the correct layer rather than load the whole map?

Reply:
Just loading the correct feature layer would certainly be useful here. My worry is that the `PortalItem` contains the  "Human Geography" basemap, which is used across other platforms. Not loading the whole map from the `PortalItem` would deviate the sample from other SDKs, and I'm curious to know if this is something worth addressing.